test/run_test

```diff
@@ -832,10 +845,18 @@ run_s2i_bake_data_test ()
 }
 
 function run_all_tests() {
+  local suite_result=0
   for test_case in $TEST_LIST; do
     : "Running test $test_case"
-    $test_case
+    if $test_case ; then
+      printf -v test_short_summary "${test_short_summary}[PASSED] $test_case\n"
+    else
+      printf -v test_short_summary "${test_short_summary}[FAILED] $test_case\n"
```

Comment:
why not:
`var="${var}..`

Reply:
ah ... the newline...  it would require `$nl` define like
```
nl='
'
test_short_summary="$test_short_summary[PASSED] $test_case$nl"

Reply:
We are in postgresql-container, in container-common-scripts this is used e.g. in generate.sh.